pubspec.lock

```diff
@@ -161,7 +161,7 @@ packages:
       name: crypto
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.4"
+    version: "2.1.3"
```

Comment:
@matthiasbruns I see all downgraded version, any reason ?

Reply:
error while rebasing. Give me a sec

Reply:
I need to find out why my cli keep downgrading the dependencies